### PR TITLE
e2e - Command results at Workbench

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/pageObjects/workbench-page.ts [hunk 1]
```diff
@@ -111,6 +117,12 @@ export class WorkbenchPage {
       await t.click(this.textViewTypeOption);
   }
 
+  //Select Tab;e view option in Workbench results
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/workbench-page.ts:120]: Table

File: tests/e2e/tests/regression/workbench/command-results.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        await userAgreementPage.acceptLicenseTerms();
+        await t.expect(addRedisDatabasePage.addDatabaseButton.exists).ok('The add redis database view', {timeout: 20000});
+        await addNewStandaloneDatabase(ossStandaloneConfig);
+        //Connect to DB
+        await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
+        //Add index and data
+        await t.click(myRedisDatabasePage.workbenchButton);
+        for(let command of commandsForIndex) {
+            await workbenchPage.sendCommandInWorkbench(command);
+        }
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/workbench/command-results.e2e.ts:35]: If all of this, of part of it, exists on other tests, let's make one function in common actions.

File: tests/e2e/tests/regression/workbench/command-results.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    })
+    .afterEach(async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/workbench/command-results.e2e.ts:37]: I couldn't find `t` usage here

File: tests/e2e/tests/regression/workbench/command-results.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        //Drop index and dbs
+        await workbenchPage.sendCommandInWorkbench('FT.DROPINDEX products DD');
+        await myRedisDatabasePage.deleteAllDatabases();
+    })
+//skipped due the inaccessibility of the iframe
+test.skip('Verify that user can switches between Table and Text for FT.INFO and see results corresponding to their views', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/workbench/command-results.e2e.ts:43]: You could set 1 skip on the fixture, instead of on each test

**Review by ofersteinberg — CHANGES_REQUESTED** [review]

File: tests/e2e/pageObjects/workbench-page.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/pageObjects/workbench-page.ts]: done

File: tests/e2e/tests/regression/workbench/command-results.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/workbench/command-results.e2e.ts]: done

File: tests/e2e/tests/regression/workbench/command-results.e2e.ts (hunk 3 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/workbench/command-results.e2e.ts]: removed

File: tests/e2e/tests/regression/workbench/command-results.e2e.ts (hunk 4 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/workbench/command-results.e2e.ts]: done

**Review by ofersteinberg — DISMISSED** [review]

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**